Code/Editor/Plugins/ProjectSettingsTool/Validators.h

```diff
@@ -24,6 +24,8 @@ namespace ProjectSettingsTool
 
         // Returns true if valid cross platform file or directory name
         FunctorValidator::ReturnType FileName(const QString& name);
+        // Returns true if valid iOS file or directory name
+        FunctorValidator::ReturnType FileNameWithPeriod(const QString& name);
```

Comment:
If this function was meant to validate iOS filenames, then it should be named something more specific
```suggestion
        FunctorValidator::ReturnType IOSFileName(const QString& name);
```

Reply:
Done.